qt/qtsystrayicon.py

```diff
@@ -56,7 +60,7 @@ def __init__(self):
         self.qapp.setQuitOnLastWindowClosed(False)
 
         import icon
-        self.icon = icon
+        self.icon = icon  # What does this code do? Make the import accessible?
```

Comment:
Yes.
It is the same in `app.py`'s MainWindow.
Let's keep it that way until next refactor session.